src/Auth0.php

```diff
@@ -594,12 +594,18 @@ public function exchange()
     /**
      * Renews the access token and ID token using an existing refresh token.
      * Scope "offline_access" must be declared in order to obtain refresh token for later token renewal.
-     *
+     * @param array $options Options for the token endpoint request.
+     *      - options.grant_type    Grant type to use; required.
+     *      - options.client_id     Application Client ID; required.
+     *      - options.client_secret Application Client Secret; required if token endpoint requires authentication.
```

Comment:
I would definitely leave these out since the first must be `refresh_token` (and is added automatically) and the others are added automatically with whatever the `Auth0` object was initiated with. The `audience` is ignored in the refresh case (that info is part of the refresh token record). And, in the case of `scope`, that's ignored in most cases as well.

Reply:
@joshcanhelp updated, please let me know if this satisfied you.